includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -43,14 +46,15 @@ public static function init() {
 		add_action( 'job_manager_init_admin_notices', [ __CLASS__, 'init_core_notices' ] );
 		add_action( 'admin_notices', [ __CLASS__, 'display_notices' ] );
 		add_action( 'wp_loaded', [ __CLASS__, 'dismiss_notices' ] );
+		add_action( 'wp_ajax_wpjm_dismiss_notice', [ __CLASS__, 'handle_notice_dismiss' ] );
 	}
 
 	/**
 	 * Add a notice to be displayed in WP admin.
 	 *
-	 * @since 1.32.0
-	 *
 	 * @param string $notice Name of the notice.
+	 *
+	 * @since 1.32.0
```

Comment:
Out of curiosity, is there any special reason to move the `@since` to the last item?

Reply:
I haven't been able to find anything suggesting the order is important.

This was done by PHPStorm's linter, and I decided to keep it as it made sense to me in terms of consistency. When I compare, for instance, with Sensei's [class-sensei-admin-notices.php](https://github.com/Automattic/sensei/blob/trunk/includes/admin/class-sensei-admin-notices.php), the `@since` isn't consistently placed in doc blocks (that I can see): sometimes it's last, sometimes it's first.

Reply:
Yeah, I would disable these "automatic" changes in your PHPStorm configuration. Imagine that VSCode would do the same but putting the since first. We would be "fighting" each other in each change.

Reply:
I reverted these @since changes in my PR [here](https://github.com/Automattic/WP-Job-Manager/pull/2398) (still in draft at the time of writing this).